src/components/Hero.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import heroHome from "../assets/images/heroHome.jpg"
+import { StaticImage } from "gatsby-plugin-image"
+import YohDevButton from "./YohDevButton"
+
+const sectionStyle = {
+  backgroundImage: `url(${heroHome})`,
```

Comment:
Can this be passed as a prop? Import it into the home.js and passed.May need GatsbyImage instead of StaticImage. This would allow us to be able to change the background image dynamically.

Reply:
I'm not sure how to solve this one since it's applying a background image as a style and not a physical image. I store the url in a variable